Content.Server/Silicons/Borgs/BorgSystem.MMI.cs

```diff
@@ -75,7 +87,10 @@ private void OnMMILinkedRemoved(EntityUid uid, MMILinkedComponent component, Ent
         RemComp(uid, component);
 
         if (_mind.TryGetMind(linked, out var mindId, out var mind))
+        {
             _mind.TransferTo(mindId, uid, true, mind: mind);
+            EnsureComp<JobComponent>(mindId).Prototype = component.OldJob;
```

Comment:
I am not familiar with `JobComponent` -- is it at all an issue if `component.OldJob` is null and the entity now has a `JobComponent` that's not meaningful because its content is empty?